src/components/member-list-item/index.js

```diff
@@ -1,11 +1,49 @@
-import React from 'react';
+import React, { useRef } from 'react';
 import Link from 'next/link';
 import PropTypes from 'prop-types';
 import Card from '@components/member-card';
 import { container } from '@components/member-card/card.module.scss';
+import { useRouter } from 'next/router';
 
 const PreviewMember = ({ memberDetails }) => {
   const { username, first_name, last_name } = memberDetails;
+  const divref = useRef(null);
+  const cardRef = useRef(null);
+  const { query } = useRouter() || { query: { dev: false } };
+  const { dev } = query;
+  const descriptionDivStyle = {
+    backgroundColor: 'rgb(0, 0, 0, 0.5)',
+    position: 'absolute',
+    display: 'none',
+    color: 'white',
+    flex: '1 250px',
+    flexDirection: 'column',
+    border: 'none',
+    borderRadius: '5%',
+    marginTop: '-1.3rem',
+    marginLeft: '-0.65rem',
```

Comment:
why negative margins?

Reply:
because I have o place it a bit above and left